input/file/inputfile.go

```diff
@@ -69,6 +69,11 @@ func InitHandler(ctx context.Context, raw *config.ConfigRaw) (config.TypeInputCo
 		return nil, err
 	}
 
+	conf.Codec, err = config.GetCodec(ctx, *raw)
+	if conf.Codec == nil {
+		conf.Codec, _ = config.DefaultCodecInitHandler(nil, nil)
```

Comment:
Please handle error everywhere even you know it will not return error when writing golang.
Because other developers may change the function in the future.